### DISCUSSION TITLE
ΓΕ4 - ΟΣΣ? για την προαιρετική παρουσίαση άρθρου

### DISCUSSION THREAD

**Discussion by afoscolou** [sdy60/syllabus#66] [Q&A] [answered]
Κύριε Χωριανόπουλε,

θα ήθελα να ρωτήσω για το μέρος Δ της ΓΕ4, την προαιρετική παρουσίαση ενός επιστημονικού άρθρου. Αναγράφεται ότι θα γίνει στην αμέσως επόμενη ΟΣΣ από την ανακοίνωσή της (πριν την τυπική προθεσμία παράδοσης της ΓΕ που είναι 29/3/23). Άρα εννοείτε την ΟΣΣ4 στις 26/3/23 ή κάποια έκτακτη προηγούμενή της; Και όχι την ΟΣΣ3 που είναι στις 11/2/23, δηλαδή 2 μέρες μετά την ανακοίνωσή της...

**Comment by epidrome** [comment, marked as answer]
Σωστά, προφανώς η προθεσμία αναφέρεται στην ΟΣΣ4.

Ενθαρρύνεται η αποστολή όσο γίνεται νωρίτερα, αφού πριν την τυπική ΟΣΣ4 μπορούμε να κάνουμε μερικές συμπληρωματικές ΟΣΣ
